README.md

```diff
@@ -46,7 +46,9 @@ If your app depends on the old unmigrated atmosphere packages than you also need
 
 ## The simplest possible test
 
-Basically, you run the tests with `gagarin` command within you project root.
+First run your meteor app with command `meteor`. 
```

Comment:
@parhelium Have you tried the latest version `0.4.6`? Also try to run `gagarin` in `--verbose` mode to get more feedback on what's going on. Please let me know about the results.